experiment/build/generate_cloudbuild.py

```diff
@@ -13,28 +13,69 @@
 # limitations under the License.
 """Generates Cloud Build specification"""
 
-import argparse
-import json
-import yaml
+import os
 
 from experiment.build import docker_images
 
+BASE_TAG = 'gcr.io/fuzzbench'
+EXPERIMENT_TAG = "${_REPO}"
+FUZZER_VAR = "${_FUZZER}"
+BENCHMARK_VAR = "${_BENCHMARK}"
+EXPERIMENT_VAR = "${_EXPERIMENT}"
 
-# TODO(Tanq16): Add unit test for this.
-def create_cloud_build_spec(buildable_images, docker_registry):
-    """Returns Cloud Build specificatiion."""
+
+def _identity(name):
+    return name.replace(FUZZER_VAR, "fuzzer").replace(BENCHMARK_VAR,
```

Comment:
I'm confused why this works.
Basically we have something like '{fuzzer}-{benchmark}-builder' in image_types.yaml.
So you replace `fuzzer` with `${_FUZZER}`. Don't you end up with `{${_FUZZER}}`?
In any case, since we are replacing {fuzzer} should you be using `.format` instead of `.replace`?

Reply:
So, the string.format method replaces '{fuzzer}' with '${_FUZZER}'. But the identity replaces (without string.format) '${_FUZZER}' with 'fuzzer' just for naming the id value for the steps.

> since we are replacing {fuzzer}

This is already done in `docker_images.py`